doc/build-netbsd.md

```diff
@@ -54,27 +52,24 @@ export BDB_PREFIX="$PWD/db4"
 With wallet:
 ```bash
 ./autogen.sh
-./configure --with-gui=no CPPFLAGS="-I/usr/pkg/include" \
-    LDFLAGS="-L/usr/pkg/lib" \
-    BOOST_CPPFLAGS="-I/usr/pkg/include" \
-    BOOST_LDFLAGS="-L/usr/pkg/lib" \
-    BDB_LIBS="-L${BDB_PREFIX}/lib -ldb_cxx-4.8" \
-    BDB_CFLAGS="-I${BDB_PREFIX}/include" \
-    MAKE=gmake
+
+MAKE="gmake" \
+CPPFLAGS="-I/usr/pkg/include" LDFLAGS="-L/usr/pkg/lib -Wl,-R/usr/pkg/lib" \
+BDB_LIBS="-L${BDB_PREFIX}/lib -ldb_cxx-4.8" BDB_CFLAGS="-I${BDB_PREFIX}/include" \
+./configure --with-gui=no --with-boost=/usr/pkg
```

Comment:
Why are you canging this around? In general we prefer putting the environment variables after the `./configure` because then they are retained if configure is automatically re-run by the build system (e.g. after a git pul lthat changes one of the build system files).

Reply:
I'm not stuck on why particular way, or order, of anything. It's entirely a habit born of porting stuff to NetBSD, idea being to keep changes as unobtrusive and light as possible, and to leave the onus on the NetBSD side to maintain/make changes that are specific to NetBSD. I can switch it up if you think that's better. I don't think any of these things will (presumably) ever be run on anything but an end-user's NetBSD machine who is doing a manual one-off build and carefully testing it by hand.

Reply:
I'd prefer to keep it as-is, it's the convention that we use everywhere and if there is no need to deviate from it on NetBSD, it's better to keep it more or less the same. This makes maintenance easier for us.

Reply:
Yeah sure thing. I'm completely fine with that. Technically no need whatsoever. Purely habit.